electron/electron.livepeer.js

```diff
@@ -18,9 +18,10 @@ const startLivepeer = (sender) => {
             global.sharedObj.livepeerProc = 'local';
             log.info('LivePeer is already running.');
         } else if (global.sharedObj.livepeerProc == null) {
-            const args = [];
+            const args = [
+                '-monitor', true,
+                '-monitorhost', 'http://viz.livepeer.org:8081/metrics'];
```

Comment:
yeah, because node expect `-monitor=` so it miss the `=`, to make it work you can 
```js
const args = [
    '-monitor=', true,
    '-monitorhost', 'http://viz.livepeer.org:8081/metrics'];
```

But I think it's more a bug on the node side, right ?